hooks/016-ensure-bootloaders.chroot

```diff
@@ -21,6 +22,5 @@ case "$(dpkg --print-architecture)" in
 EOF
         ;;
     amd64|i386)
-        mkdir /boot/efi
```

Comment:
Can we simply remove the amd64|i386 case altogether now? Doesn't make sense to leave an empty case-statement here.